### ISSUE TITLE
Improve Dockerfile

### ISSUE THREAD

**Issue by js-kyle** [UniversalMediaServer/UniversalMediaServer#2453]
As per some comments on the forum, some enhancements could be made to the Dockerfile

https://www.universalmediaserver.com/forum/viewtopic.php?f=9&t=12922

**Comment by SubJunk** [comment]
Any interest @atamariya?

**Comment by atamariya** [comment]
~/.config/UMS/UMS.conf (Only change network_interface to correspond with the one on host)

```
folders = /media
network_interface = wlp2s0

```

```
docker pull universalmediaserver/ums
docker run -it -p 5002:5002 -p 9001:9001 -v ~/Music:/media -v ~/.config:/root/.config  --network=host universalmediaserver/ums

```
I tested today and this works for me. I can browse media on my LG TV. So hopefully no change to Dockerfile. Can somebody confirm?

**Comment by gchristian** [comment]
Yes it works. I think it would be beneficial to remove the VOLUME and EXPOSE commands from the dockerfile though. Those are defined at run time anyway. And the /profile one is extra confusing because the docker image doesn't do anything with that path - you need to map to /root/.config. If the docker image also set an environment variable for UMS_PROFILE that told it to create/look for the profile in /profile then it'd make more sense for a docker volume to be created when the image is instantiated to house it. But I personally wouldn't use a docker volume for that I would do what you did there and just mount a local directory.  Also, since network=host is required for a renderer to find the server, all ports are exposed anyway.

**Comment by SubJunk** [comment]
Please feel free to contribute any improvements via pull request :)

**Comment by pataquets** [comment]
Cross-linking to related #3206.
Also, exposing the run defaults via environment vars with default (eg.` ${UMS_PROFILE:-/profile}`) would allow easily overriding them on run:
```console
$ docker run -e UMS_PROFILE=/myprofile universalmediaserver/ums
```
